components/uniset/src/props.rs

```diff
@@ -2,6 +2,9 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+//! *** Note: DO NOT USE THESE APIs FOR NOW. ****
+//!  Performance improvements and other fixes are still needed on this component.
+
```

Comment:
I'd just leave it at this comment.  I don't think you need to make any other actual changes in this PR.